app/src/main/assets/fhconfig.properties

```diff
@@ -1,5 +1,6 @@
-host = https://engdev-t.sandbox.feedhenry.com
-appid = kdwlfai7i2fiawm7a33nthaa
-projectid = kdwlfamzejuqwxg5nykpwzpp
-appkey = 279f99742ac0697b945deb4b3428b23be5ce89bb
-connectiontag = 0.0.3
\ No newline at end of file
+host = https://testing.feedhenry.me
+appid = l5z254um56q52pbqnmv5sqzf
+projectid = l5z254vsghfpliwin3z5v4zo
+appkey = 9e94fbb05e71f201754a6d1de205cb3524b6b37
+connectiontag = 0.0.8
+
```

Comment:
Let's use the old credentials so, people can use it online, or let's remove credentials.